gateway/handler.go

```diff
@@ -263,6 +263,7 @@ func (i *handler) getOrHeadHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Detect when If-None-Match HTTP header allows returning HTTP 304 Not Modified
+	// TODO: Handle If-None-Match for CAR files once what goes in the ETag is resolved
```

Comment:
@lidel any thoughts on what to do here with CAR files and what the new etag should be (although if  you feel that's a separate flag then feel free to comment on the TODO line in handler_car)

Reply:
here's what we're currently doing, if it helps, etag as stringified CID: https://github.com/filecoin-project/lassie/blob/main/pkg/server/http/ipfs.go#L190-L199

Reply:
not sure why @lidel's comment is showing up now for me, after i posted mine; from an unposted review perhaps?

anyway, human readable etag suffix would be nice, and easier across limitations; it'd be nice to lock down the form in a spec and give ourselves room to manoeuvre when we add/remove parameters. Having optional pieces of a `.` separated string is probably not a good idea for scalability so default values for missing params would be good. Is `0` OK for both `from` and `to`?

Reply:
The proposal above handles params, but doesn't say anything about the pathing. Any thoughts on that?

Previously this was easy because it was just the whole graph under a root. Now it's the path to some terminal element and then the operations (e.g. dag-scope, entity-bytes) on that.

Reply:
Pathing behavior is not configurable, we are doing what we should be always doing, so it does not impact caching /  `Etag`.

Reply:
This has been addressed.